Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
```

Comment:
I don't think we should be dropping anything from the output, no matter how popular the library is. (Yes, I realize that basically everything in OS X has to link to this particular library. That's actually not a bad thing, as this will effectively be a list of all Mach-O binaries in the keg.)